resources/language/messages.pot

```diff
@@ -2204,3 +2204,7 @@ msgstr ""
 msgctxt "#30688"
 msgid "Skip searching for installed Elementum repository"
 msgstr ""
+
+msgctxt "#30435"
```

Comment:
Usually messages go in the numeric order and for new messages we take new values, not used before, to make sure it will not overwrite anything in the code.

Reply:
@elgatito i used `xgettext_get_available_ids.sh` (that i wrote for such cases) to find available id. but i can change this 30435 to 30688 if you like.

Reply:
There are already messages up to `30690` :)

Please, use increments from latest available.

Reply:
@elgatito done. git is really bad with merging po files.